app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -334,33 +333,16 @@ class ComposeViewModel @Inject constructor(
                 }
             }
         }
-
-        if (!editing) {
-            // Updates to media for already-published statuses need to go through the status edit api
```

Comment:
I wonder if this comment was true even before?

With this new code (in StatusSendService) it also works when editing a status.

(maybe @mcclure ?)

Reply:
I don't recognize this code, sorry. The only advice I have is to make sure your change works when editing (cropping) an image (because that at some level deletes the image and replaces it with a new one)

Reply:
It was _absolutely_ true before - without this codepath, when editing the caption of an attachment to an already-published status (post status with media attachment, edit status, edit caption of attachment), the user would get a popup about setting the caption failing

Reply:
https://github.com/tuskyapp/Tusky/pull/3215#issuecomment-1407476422

Reply:
The description/focus *do* get updated, but because of the subsequent call to `editStatus`, not the `updateMedia` call here, which fails.

Reply:
Very odd that the call to update media is then necessary initially.

But you are right.

Added "isEditing".